input-image/image/image.css

```diff
@@ -0,0 +1,10 @@
+@import url("/style.css");
+
+.round-img{
+    height: 8rem;
+    width: 5rem;
```

Comment:
Suggestion: You can use same width and height to make it aa square and then apply border-radius to 50%. this will result in a complete rounded figure. Right now it looks a bit oval-shaped.